calico-enterprise_versioned_docs/version-3.19-1/releases.json

```diff
@@ -1,247 +1,269 @@
 [
   {
-    "title": "master",
+    "title": "v3.19.0-1.0",
     "tigera-operator": {
       "image": "tigera/operator",
-      "version": "master",
+      "version": "v1.31.2",
       "registry": "quay.io"
     },
     "calico": {
-      "minor_version": "v3.26",
+      "minor_version": "v3.27",
       "archive_path": "archive"
     },
     "components": {
       "cnx-manager": {
         "image": "tigera/cnx-manager",
-        "version": "master"
+        "version": "v3.19.0-1.0"
       },
       "voltron": {
         "image": "tigera/voltron",
-        "version": "master"
+        "version": "v3.19.0-1.0"
       },
       "guardian": {
         "image": "tigera/guardian",
-        "version": "master"
+        "version": "v3.19.0-1.0"
       },
       "cnx-apiserver": {
         "image": "tigera/cnx-apiserver",
-        "version": "master"
+        "version": "v3.19.0-1.0"
       },
       "cnx-queryserver": {
         "image": "tigera/cnx-queryserver",
-        "version": "master"
+        "version": "v3.19.0-1.0"
       },
       "cnx-kube-controllers": {
         "image": "tigera/kube-controllers",
-        "version": "master"
+        "version": "v3.19.0-1.0"
       },
       "calicoq": {
         "image": "tigera/calicoq",
-        "version": "master"
+        "version": "v3.19.0-1.0"
       },
       "typha": {
         "image": "tigera/typha",
-        "version": "master"
+        "version": "v3.19.0-1.0"
       },
       "calicoctl": {
         "image": "tigera/calicoctl",
-        "version": "master"
+        "version": "v3.19.0-1.0"
       },
       "cnx-node": {
         "image": "tigera/cnx-node",
-        "version": "master"
-      },
-      "cnx-node-windows": {
-        "image": "tigera/cnx-node-windows",
-        "version": "master"
+        "version": "v3.19.0-1.0"
       },
       "dikastes": {
         "image": "tigera/dikastes",
-        "version": "master"
+        "version": "v3.19.0-1.0"
       },
       "dex": {
         "image": "tigera/dex",
-        "version": "master"
+        "version": "v3.19.0-1.0"
       },
       "fluentd": {
         "image": "tigera/fluentd",
-        "version": "master"
+        "version": "v3.19.0-1.0"
       },
       "fluentd-windows": {
         "image": "tigera/fluentd-windows",
-        "version": "master"
+        "version": "v3.19.0-1.0"
       },
       "es-proxy": {
         "image": "tigera/es-proxy",
-        "version": "master"
+        "version": "v3.19.0-1.0"
       },
       "eck-kibana": {
-        "version": "7.17.9"
+        "version": "7.17.13"
       },
       "kibana": {
         "image": "tigera/kibana",
-        "version": "master"
+        "version": "v3.19.0-1.0"
       },
       "eck-elasticsearch": {
-        "version": "7.17.9"
+        "version": "7.17.13"
```

Comment:
Should be 7.17.14 according to tigera/operator/config/enterprise_versions.yml

Reply:
Fixed.